### ISSUE TITLE
`__pragma("warning")`?

### ISSUE THREAD

**Issue by Daniel-Cortez** [Zeex/pawn#588]
### Issue description:
Since we already have `#pragma warning` as a directive, we could also have `__pragma("warning ...")`. I think this would be especially useful for macros, so warnings could be temporarily enabled/disabled from there.

### Minimal complete verifiable example (MCVE):
```Pawn
// Let's suppose that for some reason we need to make a passed-by-reference function argument constant.
// Usually the compiler prints "warning 238: meaningless combination of class specifiers" in such cases,
// so we'll need to temporarily disable this warning.
Func(const __pragma("warning push", "warning disable 238") &arg __pragma("warning pop"))) {}
```
### Workspace Information:

* Compiler version:
* Command line arguments provided (or sampctl version):
* Operating System:

**Comment by Y-Less** [comment]
Yes

**Comment by Daniel-Cortez** [comment]
Implemented in #589, closing.
